src/pages/SearchPage/ui/SearchPage.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component } from 'react';
+import { ItemCard } from '@entities/ItemCard';
+import { Item } from '@shared/api/items';
+import { Loader } from '@shared/ui/Loader/Loader';
+import css from './style.module.css';
+import '@shared/styles/global.css';
+
+interface Props {
+  items: Item[];
+  isFetching: boolean;
+  isResultEmpty: boolean;
+}
+export class SearchPage extends Component<Props> {
+  renderItems() {
+    return this.props.items.map((item: Item) => (
+      <ItemCard
+        key={item.mal_id}
+        mal_id={item.mal_id}
+        title={item.title}
+        year={item.year}
+        duration={item.duration}
+        episodes={item.episodes}
+        favorites={item.favorites}
+        images={item.images}
+      />
```

Comment:
Why not to pass just item into this component? Does it make sense to pass all the data property by property?
```suggestion
      <ItemCard
        cardData={item}
      />
```